controller/controller_server/utils.py

```diff
@@ -96,6 +98,14 @@ def _validate_source_info(source, source_type):
         raise ObjectIdError(source_type, source_object_id)
 
 
+def _validate_create_volume_parameters(parameters):
```

Comment:
what about non-pool parameters?

Reply:
we have only space efficiency except of pool

Reply:
how do you know? and even if so, why isn't it part of this function?

Reply:
I read our code, its in [here](https://github.com/IBM/ibm-block-csi-driver/blob/7adaecee45fe926ab778450721836d46a0d7c10e/controller/controller_server/csi_controller_server.py#L405)

Reply:
your link points only to volume capabilities portion of the parameters (it doesn't even cover the pool parameter).
could you point to where you concluded that we only have two parameters?

Reply:
done (links in [here](https://github.com/IBM/ibm-block-csi-driver/pull/212#discussion_r551876581))

Reply:
nice, and I see you also renamed this to `_validate_pool_parameter` - great!